File: guanabara-java/src/com/danillo/algoritmos/lista02/Exercicio18.java
package com.danillo.algoritmos.lista02;

import java.util.Scanner;

public class Exercicio18 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o ano em que você nasceu: ");
        int anoNascimento = entrada.nextInt();

        int anoAtual = 2024;
        int idade = anoAtual - anoNascimento;
        if (idade >= 16){
            System.out.println("Você tem " +idade+ " anos e pode votar.");
        } else {
            System.out.println("Você tem " +idade+ " anos e ainda não pode votar.");
        }
    }
}

File: guanabara-c/lista02/exercicio18.c
#include <stdio.h>

int main(){
    int ano_atual, ano_nascimento, idade;

    printf("Digite o ano atual: ");
    scanf("%d", &ano_atual);
    printf("Digite o ano de nascimento: ");
    scanf("%d", &ano_nascimento);

    idade = ano_atual - ano_nascimento;
    if(idade >= 16){
        printf("Idade: %d anos. Você pode votar.", idade);
    }
    else{
        printf("Idade: %d anos. Você não pode votar.", idade);
    }

    return 0;
}
